gmail/data_mappings.bal

```diff
@@ -24,7 +24,7 @@ type mapJson map<json>;
 # Transforms JSON message object into Message Type Object.
 # + sourceMessageJsonObject - `json` message object
 # + return - Returns Message type object
-isolated function convertJSONToMessageType(json sourceMessageJsonObject) returns @tainted Message {
+public isolated function convertJSONToMessageType(json sourceMessageJsonObject) returns @tainted Message {
```

Comment:
We kept them private as we did not expose them to the user. Cannot we keep it unchanged?

Reply:
Made it public because i wanted to use the same function in listener module also